glossary.md

```diff
@@ -13,52 +13,73 @@ gh: specifically for github
 
 
 
 ### Updating and Sharing
-Before I get into syntax, let's address definitons: 
+Before I get into syntax, let's address definitons. We talked about staging areas being a tempoary holder: 
 
-**add**: Stages a change for the next commit. You're basically marking the files as ready. 
+**add**: You add the changed files into your staging area.  
 
-**commit**: Saves the staged changes to your LOCAL repository. It creates a snapshot of the file in it's current statement
+**commit**: Saves the staged changes to your LOCAL repository. It creates a *snapshot* of the repository in it's current statement
 
-**push**: Sends your local committed changes to the remote repository.
+**push**: Sends your local committed changes to the remote repository. The staging area is now empty.
 
 
 | command    | use |
 | -------- | ------- |
-| git remote add [alias] [url]| add a git url as an alias. The alias is usually 'origin'    |
-| git fetch [alias] |fetch down all the branches from the git remote. Retreive updates from a remote repo but doesn't merge them into your current working branch. You can review changes before integrating them     |
-| git merge [alias]/[branch] | combines changes from one branch to another branch |
-| git pull | takes all commits from a remote repository and integrates them into yur current branch | 
-| git push [alias][branch] | sends local branch commits to remote repository branch | 
-| git commit -m "[message]" | commits your current content as a new commit with a message to add context |
+| git remote add [alias] [url]| This command lets you connect your personal code to the shared repository. The 'git remote add' part tells Git you want to add a new connection to the remote repository with the nickname '[alias]'. The URL is the location of the shared repository. |
+| git fetch [alias] |Let's say your friend updated the repo and you want to review the changes before putting them in your code. You use 'git fetch' to get the latest updates from the remote repo and the '[alias]' is the same nickname from when you added the repository in the previous command. However, git fetch DOES NOT add the changes to your code automatically.     |
+| git merge [alias]/[branch] | Let's say you reviewed their code and you're ready to integrate that code to your code. Git merge does just that-- it adds the changes to your code.  |
+| git pull | This basically does the work of git fetch and git merge in one command. It's when you know fosho that your friends code is good and you want to add it to your repository. | 
+|git add [filename] | This command adds the specific changed file you're referencing to in [filename] to the staging area. You're adding your new notes/changes into the tempoary section of the binder. |
+|git add . | This commands all the files that you changed to the staging area. The '.' is what makes it ALL the files you've changed | 
```

Comment:
list it, maybe tell the reader to avoid it. I'd like to avoid new devs mass adding random stuff to our repos